README.md

```diff
@@ -0,0 +1,308 @@
+# Greenfield BSC Archiver
+
+## Description
+
+The Greenfield BSC Archiver is tasked with storing headers and bodies of BSC historical blocks. It uploads this block information to Greenfield via the bundle service. The components of this archiver include:
+
+### Syncer
+
+Responsible for fetching block information via Ethereum calls, storing it in a database, and uploading it to Greenfield through the bundle service. The current setting uploads data every 100 blocks.
+
+### Verifier
+
+Regularly checks the integrity of uploaded data on Greenfield. If errors are found, it corrects and re-uploads the data, and updates the block bundle name in the database.
+
+### Server
+
+Provides RPC interfaces compatible with Ethereum calls. Variables are generated using a Swagger.yaml file to ensure adaptability.
+1. eth_getBlockByNumber
+2. eth_getBundledBlockByNumber
+3. eth_blockNumber
+4. eth_getBlockByHash
+5. /bsc/v1/blocks/{block_id}/bundle/name
```

Comment:
TODO: put the public endpoint of the service after deploy

Reply:
add the endpoint https://gnfd-bsc-archiver-testnet.bnbchain.org/ to the readme